CefSharp.BrowserSubprocess.Core/CefAppUnmanagedWrapper.cpp

```diff
@@ -60,22 +60,19 @@ namespace CefSharp
     {
         auto browserWrapper = FindBrowserWrapper(browser->GetIdentifier(), true);
 
-        if (!Object::ReferenceEquals(_javascriptRootObject, nullptr) || !Object::ReferenceEquals(_javascriptAsyncRootObject, nullptr))
-        {
-            auto rootObjectWrappers = browserWrapper->JavascriptRootObjectWrappers;
-            auto frameId = frame->GetIdentifier();
+        auto rootObjectWrappers = browserWrapper->JavascriptRootObjectWrappers;
+        auto frameId = frame->GetIdentifier();
 
-            if (rootObjectWrappers->ContainsKey(frameId))
-            {
-                LOG(WARNING) << "A context has been created for the same browser / frame without context released called previously";
-            }
-            else
-            {
-                auto rootObject = gcnew JavascriptRootObjectWrapper(browser->GetIdentifier(), browserWrapper->BrowserProcess);
-                rootObject->Bind(_javascriptRootObject, _javascriptAsyncRootObject, context->GetGlobal());
+        if (rootObjectWrappers->ContainsKey(frameId))
+        {
+            LOG(WARNING) << "A context has been created for the same browser / frame without context released called previously";
+        }
+        else
+        {
+            auto rootObject = gcnew JavascriptRootObjectWrapper(browser->GetIdentifier(), browserWrapper->BrowserProcess);
+            rootObject->Bind(_javascriptRootObject, _javascriptAsyncRootObject, context->GetGlobal());
```

Comment:
I'd consider moving the `if (!Object::ReferenceEquals(_javascriptRootObject, nullptr) || !Object::ReferenceEquals(_javascriptAsyncRootObject, nullptr))` check here, so we don't bother calling bind or getting a context. And that's just a minor tweak :+1: